qiskit_experiments/characterization/qubit_spectroscopy.py

```diff
@@ -208,14 +228,37 @@ def _default_run_options(cls) -> Options:
         )
 
     @classmethod
-    def _default_experiment_options(cls) -> Options:
+    def _default_experiment_options(cls) -> Union[Options, Dict[str, OptionsField]]:
         """Default option values used for the spectroscopy pulse."""
-        return Options(
-            amp=0.1,
-            duration=1024,
-            sigma=256,
-            width=0,
-        )
+        return {
+            "amp": OptionsField(
+                default=0.1,
+                annotation=float,
+                description="Amplitude of spectroscopy pulse. Usually weak power pulse is used to "
+                "suppress broadening of observed peaks.",
+            ),
+            "duration": OptionsField(
+                default=1024,
+                annotation=int,
+                description="Duration of spectroscopy pulse. This may need to satisfy the "
+                "hardware waveform memory constraint. "
+                "The default value is represented in units of dt.",
+            ),
+            "sigma": OptionsField(
+                default=256,
+                annotation=Union[int, float],
+                description="Sigma of Gaussian rising and falling edges. This value should be "
+                "sufficiently smaller than the duration, "
+                "otherwise waveform is distorted. "
```

Comment:
```suggestion
                description="Sigma of Gaussian rising and falling edges."
```
We do not need to enforce/suggest this, you can also run spectroscopy with a long pure Gaussian pulse and get good results.